### PR TITLE
Add DAO to vote and stake

### LINKED ISSUE #65: proposalVote and ProposlStake are not filterable by DAO
We will (not immediately but at a certain point) need all proposalVotes associated with a particular DAO. 

In other words, `proposalVotes` could use an option to filter by dao address...

### LINKED ISSUE #66: proposalStakes are not filterable by DAO
We will need this at some point. Issue #65  is directly related to this one

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #65 
Fix #66

### CODE REVIEW

File: src/domain/stake.ts [hunk 1]
```diff
@@ -19,13 +19,15 @@ export function insertStake(
   staker: Address,
   amount: BigInt,
   proposalId: string,
+  daoId: string,
```
Comment (orenyodfat) [inline src/domain/stake.ts]: why it is daoId and not dao ?

Comment (ben-kaufman) [reply src/domain/stake.ts]: Same reason the above is proposalId and not proposal. You don't need the object just the ID.

Comment (orenyodfat) [reply src/domain/stake.ts]: why not the object ?

**Comment by orenyodfat** [comment]
Does this cannot be achieved by queuring proposalVotes using 'where' with a filter ?
 something like proposalVotes where proposal dao equal to dao Id ?

**Comment by ben-kaufman** [comment]
I'm pretty sure that's not possible.

**Pull request merged**